scaffolding-chef-inspec/tests/linux/user-a2insecure-profile-dep/controls/example.rb

```diff
@@ -0,0 +1,5 @@
+title 'CIS Automate Test'
```

Comment:
Let's talk more about this but these are not the right test files that need to be updated. The test that need to be added for this are the ones that run through expeditor.